packages/langchain-graph-retriever/src/langchain_graph_retriever/transformers/parent.py

```diff
@@ -0,0 +1,73 @@
+from collections.abc import Sequence
+from typing import Any
+
+from langchain_core.documents import BaseDocumentTransformer, Document
+from typing_extensions import override
+
+
+class ParentTransformer(BaseDocumentTransformer):
```

Comment:
Nice. I think it's useful to have this transformer (especially for LangFlow, etc.) but this could be a good thing to have an example of doing too. Specifically, if the user has already written code to put the path in the metadata... it wouldn't be too hard to just write the parent themselves.

We should definitely have some examples showing "here is how to set the metadata for X" rather than relying on hiding it all in transformers. Specifically, that enables the *user* to encode their own structured metadata.

This would be a good example to use, since it demonstrates "setting something like `parent` in the metadata enables different kinds of navigation via the edges".